pkg/apis/tensorflow/v1/types.go

```diff
@@ -69,6 +69,13 @@ type TFJobSpec struct {
 	//     "Worker": ReplicaSpec,
 	//   }
 	TFReplicaSpecs map[TFReplicaType]*common.ReplicaSpec `json:"tfReplicaSpecs"`
+
+	// TFRpcLayer is rpc layer in TFConfig
+	// For example,
+	// {
+	//    "rpcLayer": TFRpcLayer
+	// }
+	TFRpcLayer string `json:"tfRpcLayer,omitempty"`
```

Comment:
Can we define it as *string?

Reply:
I think we can define it as *string. 
But in that case, I think zz-generated.deepcopy.go should be regenerated.
I don't know how to regenerated it.
Only I can do is add the following code into it
```
if in.TFRPCLayer != nil {
		in, out := &in.TFRPCLayer, &out.TFRPCLayer
		*out = new(string)
		**out = **in
	}
```

Reply:
You can run ./hack/update-codegen.sh

Reply:
I installed the latest openapi-gen. And it update lots of code. Is it ok?

Reply:
Yeah, I think so.